db/migrate/20190514124322_create_firmware_binaries.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFirmwareBinaries < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmware_binaries do |t|
+      t.string     :name
+      t.string     :external_id
```

Comment:
Looks like the specs are failing because it is expecting any `_id` to be a bigint.  How about `:external_ref` here instead?